Created a basic starter iPhone app

goranche opened the pull request:
depends on cocoaheads-slovenija/theClocker#12

goranche committed:
Created a basic starter iPhone app

goranche committed:
Changed model from struct to class

adelburekovic commented on chargeOclock/Source/DataSources/ClientsDataSource.swift:
```diff
@@ -0,0 +1,37 @@
+class ClientsDataSource: NSObject, UITableViewDataSource {
+
+	let clients: [Client]
+	let reuseIdentifier: String
```
I know it's little early for things like this. But how will we handle access levels in this project? Does reuseIdentifier for instance really need to be internal?

goranche replied:
it doesn't need to, but since it's a constant that's provided by the code instantiating this class, it doesn't hurt either and might come in handy (especially since the reuseIdentifier has a default value in the constructor, so the calling code might not even be passing it in)

goranche committed:
Changed the baseURL for the service

In addition, the server started returning ids as numbers, which is
different from what I saw locally 🤷‍♂️

adelburekovic commented on chargeOclock/Source/DataSources/ClientsDataSource.swift:
```diff
@@ -0,0 +1,37 @@
+//  chargeOclock
+//
+//  Created by Goran Blažič on 08/01/2017.
+//  Copyright © 2017 goranche.net. All rights reserved.
```
Should organization name be changed to something more appropriate? Thoughts?

goranche replied:
👍  will change to "CocoaHeads Slovenia"

adelburekovic commented on chargeOclock/Source/Other/Errors.swift:
```diff
@@ -0,0 +1,19 @@
+
+import Foundation
+
+enum oClockErrors: Error {
```
Shouldn't enumeration definitions be singular?

goranche replied:
👍 they should...

goranche committed:
Changes organisation name on project

goranche committed:
Changed error enum to singular naming
